src/main/java/org/apereo/openequella/adminconsole/service/JsonService.java

```diff
@@ -24,12 +24,12 @@
 import java.io.InputStream;
 import java.io.OutputStream;
 
-import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.json.JsonMapper;
 
 public class JsonService {
 	
-	private static final ObjectMapper jsonMapper = new ObjectMapper();
-	
+	private static final JsonMapper jsonMapper = new JsonMapper();
```

Comment:
Why the change to explicit `JsonMapper`? Unless we're need something specifically from the child implementation we normally keep a level of abstraction and go with the parent - as was there before. Further, you'll note this is the approach taken in most [documentation and tutorials](https://github.com/FasterXML/jackson-docs#tutorials).

Unless there is a need, could we please change back?

Reply:
After reading the release notes / change log, I had thought it was more secure.  However on re-reviewing the documentation, I only found a specific reference (below), but I can't go so far as to say it's a need.  I'll flip it back.

From https://github.com/FasterXML/jackson-databind/issues/2153 :
> ObjectMapper will still exist, and may even be instantiated (more so with 2.10, less with 3.x), but usage should move over to specific mappers for 3.x.

Reply:
Ok, that's changed back now.